cmd/PolicyGenerator/main.go

```diff
@@ -2,20 +2,48 @@ package main
 
 import (
 	"bytes"
+	_ "embed"
 	"fmt"
 	"os"
+	runtimeDebug "runtime/debug"
 
 	"github.com/spf13/pflag"
 	"open-cluster-management.io/policy-generator-plugin/internal"
 )
 
+//go:embed VERSION.txt
+var version []byte
 var debug = false
 
+// Gather the Git commit
+var gitCommit = func() string {
+	if info, ok := runtimeDebug.ReadBuildInfo(); ok {
+		for _, setting := range info.Settings {
+			if setting.Key == "vcs.revision" {
+				return setting.Value
+			}
+		}
+	}
+
+	return ""
+}()
+
 func main() {
 	// Parse command input
 	debugFlag := pflag.Bool("debug", false, "Print the stack trace with error messages")
+	versionFlag := pflag.Bool("version", false, "Print the version of the generator")
 	pflag.Parse()
 
+	if *versionFlag {
+		versionStr := string(version)
```

Comment:
Could you strip any new lines on this? I could see `VERSION.txt` have one by accident.